test/automation/src/settings.ts

```diff
@@ -56,20 +57,33 @@ export class SettingsEditor {
 
 	async openUserSettingsUI(): Promise<void> {
 		await this.quickaccess.runCommand('workbench.action.openSettings2');
-		await this.code.waitForElement('.settings-editor');
+		await this.code.waitForActiveElement(SEARCH_BOX);
 	}
 
 	async searchSettingsUI(query: string): Promise<void> {
 		await this.openUserSettingsUI();
-		await this.code.waitAndClick('.settings-editor .suggest-input-container .monaco-editor textarea');
-		if (process.platform === 'darwin') {
-			await this.code.dispatchKeybinding('cmd+a');
-		} else {
-			await this.code.dispatchKeybinding('ctrl+a');
+
+		// Retry the following in case the Settings editor reloads halfway in between,
+		// such as with https://github.com/microsoft/vscode/issues/190896
+		let tries = 0;
+		while (true) {
+			try {
```

Comment:
Retries are supported by the test runner:

https://github.com/microsoft/vscode/blob/548c32e80ca9744544a6d245e679cf14f3e3a025/test/smoke/src/areas/terminal/terminal-input.test.ts#L28-L32

If you do this there needs to be a good reason because flakiness likely means there is an actual bug.

Reply:
Reverting so that it's just the delay for now.